keyword_clustering.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import csv
+import os
+
+"""Cluster keywords based on the result of summarization."""
+
+
+def main(args):
+    input_file_path = os.path.expanduser(args.input_file_path)
+    if not os.path.isfile(input_file_path):
+        raise ValueError("Cannot find the input file.")
+
+    shortened_keywords_list = []
+    total_keyword_counts = 0
+    
+    with open(input_file_path) as f:
+        read_tsv = csv.reader(f, delimiter="\t") 
+        model_name_list = next(read_tsv)[1:]
```

Comment:
reduce one indent level: 

with open(input_file_path) as f:
        read_tsv = csv.reader(f, delimiter="\t") 
model_name_list = next(read_tsv)[1:]
for i in range(len(model_name_list)):
      shortened_keywords_list.append({})
...

Reply:
Done.